### PR TITLE
CLI: add dbms-plugin commands

### PR DESCRIPTION
**Pull request by nglgzz**
### Please check if the PR fulfills these requirements
- [x] The commit message follows our [guidelines](https://github.com/neo4j-devtools/relate/blob/master/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?
Feature

### What is the current behavior?

No CLI commands for plugins.

### What is the new behavior?

Adds the following commands:

-  `dbms-plugin:add-sources`     Add one or more plugin sources
-  `dbms-plugin:install`         Install plugin on one or more Neo4j DBMSs
-  `dbms-plugin:list`            List plugins installed in a Neo4j DBMS
-  `dbms-plugin:list-sources`    List available plugin sources
-  `dbms-plugin:remove-sources`  Remove one or more plugin sources

### Does this PR introduce a breaking change?
No

### CODE REVIEW

File: packages/cli/src/modules/dbms-plugin/add-sources.module.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        if (argv.length >= 1) {
+            const sourcesList = await List.from(argv)
+                .mapEach(async (pathOrUrl) => {
+                    try {
+                        const source = await fse.readJSON(pathOrUrl);
+                        return source;
+                    } catch {
+                        return fetch(pathOrUrl).then((res) => res.json());
+                    }
+                })
```
Comment (nglgzz) [inline packages/cli/src/modules/dbms-plugin/add-sources.module.ts:39]: Not sure what the expected behavior should be here. Should each file/URL contain only one source, or should it contain a list of sources?

File: packages/cli/docs/dbms-plugin.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  -x, --extended                 show extra columns
+  --columns=columns              only show provided columns (comma-separated)
+  --filter=filter                filter property by partial string matching, ex: name=foo
+  --no-header                    hide table header from output
+  --no-truncate                  do not truncate output to fit screen
+  --output=csv|json|yaml         output in a more machine friendly format
+  --sort=sort                    property to sort by (prepend '-' for descending)
+
+ALIASES
+  $ relate plugin:add-sources
```
Comment (jk05) [inline packages/cli/docs/dbms-plugin.md:34]: could there be other (i.e non dbms) plugins in future? I know it's just an alias, just wondering.

Comment (nglgzz) [reply packages/cli/docs/dbms-plugin.md:34]: In theory yes, in practice we can (and in my opinion should) decide to call them something else instead (in this PR I suggested "components" for example https://github.com/neo4j-devtools/relate/pull/250).

Comment (jk05) [reply packages/cli/docs/dbms-plugin.md:34]: maybe something to circle back to later

**Review by jk05 — APPROVED** [review]

**Comment by wheresrhys** [comment]
Can't wait for this to be released 😻

**Comment by nglgzz** [comment]
[[Kick-off] relate handling DBMS plugins](https://trello.com/c/LDjRztj5/1569-kick-off-relate-handling-dbms-plugins)

**Pull request merged**